pages/calendar.js

```diff
@@ -5,7 +5,7 @@ export default function Calendar() {
   return (
     <>
       <BackArrow link="/" />
-      <CalendarReminder></CalendarReminder>;
+      <CalendarReminder></CalendarReminder>
```

Comment:
This can be a self closing tag if there are no children. 

```suggestion
      <CalendarReminder />
```